src/docs/designs/hypothesis.md

```diff
@@ -8,5 +8,6 @@ relative_path: ../../../
 | <a href="javascript:SortTable(0);" id="designTableTitle" class="sort">Design | <a href="javascript:SortTable(1);" id="designTableType" class="sort">Type</a> | <a href="javascript:SortTable(2);" id="designTableDesigner" class="sort">Designer</a> | <a href="javascript:SortTable(3);" id="designTableStatus" class="sort">Status<a/> | <a href="javascript:SortTable(4, 'D', 'mdy');" id="designTableUpdate" class="sort">Last Updated</a> | <span id="designTableWILinks">Work Item Link</span>
 | :---                                              |:---                   |:---           |:---               |:---               |:---                                                                                                      |:---
 |[Prelim Wireframes](https://redhat.invisionapp.com/share/XMCK7FO6R#/243441828_Access_Hypothesis)                |Wireframes              |Laura         |In Progress      |07/12/2017         |[UX GitHub Issue 532](https://github.com/fabric8-ui/fabric8-ux/issues/532)      |
+| [Hypothesis Engine](https://drive.google.com/open?id=1jYGo2zXqSCiyCsBCDtU2oQlmLR4OodfY)             | IA + IX               | Samir    | In Progress       | 10/01/2018  |
```

Comment:
Is this an internal only link?

Reply:
This is what I got from Samir

Reply:
@mindreeper2420 @SMahil 
Any HDD related link is internal .
Please do not include them to external sites.